src/prefect/server/api/automations.py

```diff
@@ -244,14 +244,16 @@ async def read_automations(
     sort: AutomationSort = Body(AutomationSort.NAME_ASC),
     limit: int = LimitBody(),
     offset: int = Body(0, ge=0),
+    automations: Optional[AutomationFilter] = None,
     db: PrefectDBInterface = Depends(provide_database_interface),
 ) -> Sequence[Automation]:
     async with db.session_context() as session:
-        return await automations.read_automations_for_workspace(
+        return await automations_models.read_automations_for_workspace(
             session=session,
             sort=sort,
             limit=limit,
             offset=offset,
+            automation_filter=automations,
```

Comment:
Needed to rename the automations module since otherwise adding it as a param here overrides the endpoint logic below